fastlane/ScreenshotFastfile

```diff
@@ -86,7 +86,7 @@ platform :android do
     # Allow creating screenshots for just one locale
     if options[:locale] != nil
       locales.keep_if { |locale|
-        locale.casecmp(options[:locale]) == 0
+        locale != nil && locale.casecmp(options[:locale]) == 0
```

Comment:
Instead of this, can we do the following instead:

```
locales = ALL_LOCALES
      .select { |hsh| hsh[:promo_config] != false }
      .map { |hsh| hsh[:google_play] }
      .compact
```

https://ruby-doc.org/core-2.7.0/Array.html#method-i-compact